config/default.go

```diff
@@ -0,0 +1,150 @@
+package config
+
+// DefaultValues is the default fonfigurations for the hermez node
+const DefaultValues = `
+[API]
+Address = "localhost:8086"
+Explorer = true
+UpdateMetricsInterval = "10s"
+UpdateRecommendedFeeInterval = "10s"
+MaxSQLConnections = 100
+SQLConnectionTimeout = "2s"
+
+[PriceUpdater]
+Interval = "60s"
+Priority = "bitfinexV2,CoinGeckoV3"
+Statictokens="3=1.31,4=1.01"  # <tokenId>=<forced_price>,<tokenId>=<forced_price>
+
+[PriceUpdater.Fiat]
+APIKey=""
+URL="https://api.exchangeratesapi.io/v1/"
+BaseCurrency="USD"
+Currencies="CNY,EUR,JPY,GBP"
+
+[[PriceUpdater.Provider]]
+Provider = "bitfinexV2"
+BASEURL = "https://api-pub.bitfinex.com/v2/"
+URL = "ticker/t"
+URLExtraParams = "USD"
+Symbols = "0=ETH,2=UST,8=ignore,9=SUSHI:,5=WBT,14=LINK:,12=AAVE:,7=XAUT:,10=COMP:,6=ETH"
+
+[[PriceUpdater.Provider]]
+Provider = "CoinGeckoV3"
+BASEURL = "https://api.coingecko.com/api/v3/"
+URL = "simple/token_price/ethereum?contract_addresses="
+URLExtraParams = "&vs_currencies=usd"
+Addresses="0=0x0000000000000000000000000000000000000000,2=0xdac17f958d2ee523a2206206994597c13d831ec7,8=ignore"
+
+[Debug]
+APIAddress = "0.0.0.0:12345"
+MeddlerLogs = true
+GinDebugMode = true
+
+[StateDB]
+Path = "/tmp/iden3-test/hermez/statedb"
```

Comment:
It should use the same values than defined at heznode.service. Please check the `make install`  and with @mfcastellani

Reply:
```suggestion
Path = "/var/hermez/statedb"
```